docs/src/projects/delete-project.md

```diff
@@ -36,4 +36,11 @@ title=Using the CLI
 {{< /codetabs >}}
 
 You are billed only for the portion of a month when the project was active.
-A user account with no projects has no charges.
+
+When you remove all your projects from your user account,
+your subscription is canceled and you're not charged.
+To permanently delete your Platform.sh account,
+[contact Support](https://console.platform.sh/-/users/~/tickets/open).
```

Comment:
:lady_beetle: This seems like it doesn't really fit here, though I'm not sure where it should fit. Perhaps it needs a new section somewhere else?